spec/registry.md

```diff
@@ -90,7 +90,7 @@ For the sake of brevity, only `locales="en"` is considered.
     <function name="number">
         <description>
             Format a number.
-            Match a numerical value against CLDR plural categories or against a number literal.
+            Match a **formatted** numerical value against CLDR plural categories or against a number literal.
```

Comment:
Does the imply that the input is a localized number that'd be parsed and then matched? Also since this function includes both selection and formatting, should this also say something about the formatting part?

Reply:
No. I've merged a suggestion from Addison which does not mention formatting.

There is an ongoing discussion in issue https://github.com/unicode-org/message-format-wg/issues/425
Especially this comment https://github.com/unicode-org/message-format-wg/issues/425#issuecomment-1640648720